src/sentry/services/hybrid_cloud/organization/service.py

```diff
@@ -105,6 +107,46 @@ def check_membership_by_id(
         """
         pass
 
+    @regional_rpc_method(resolve=ByOrganizationIdOrSlug())
```

Comment:
This is an antipattern. Please split the method into public `get_invite_by_id` and `get_invite_by_slug` methods with non-optional parameters.

On the implementation class, they can both funnel into a common, private `_get_invite` method.

Reply:
Got it, made the change in next pass.